scout/commands/update/panel.py

```diff
@@ -46,10 +60,46 @@ def panel(panel, version, update_date, update_version):
             LOG.warning(err)
             click.Abort()
 
+    # Any mintainer updates?
+
+    new_maintainer = None
+    if add_maintainer:
+        if add_maintainer not in [user["_id"] for user in adapter.users()]:
+            # Check if maintainers exist in the user database
+            raise SyntaxError(
```

Comment:
I would prefer a `LOG.warning` and `click.Abort` or just `return`

Reply:
Sure; just following the nearest example.

Reply:
I made all of them click.Abort now. Hope it works!